### PR TITLE
Fixes for examples using bounding boxes when testing with asdf-astropy

### PR DESCRIPTION
**Pull request by WilliamJamieson**
Fixes example issues found when working on astropy/asdf-astropy#69

### CODE REVIEW

File: resources/stsci.edu/schemas/property/compound_bounding_box-1.0.0.yaml [hunk 1]
```diff
@@ -19,94 +19,141 @@ examples:
...
+          - !transform/shift-1.2.0
+            offset: 3.0
+        bounding_box: !transform/property/compound_bounding_box-1.0.0
+          selector_args:
+            - argument: x0
+              ignore: true
+            - argument: x1
+              ignore: true
+          cbbox:
+            - key: [0, 1] # both value of input x is 0 and value of input y is 1 to select this box
```
Comment (perrygreenfield) [inline resources/stsci.edu/schemas/property/compound_bounding_box-1.0.0.yaml:87]: shouldn't the comment match the `selector_arg` names? And likewise for the subsequent comments?

File: resources/stsci.edu/schemas/property/compound_bounding_box-1.0.0.yaml [hunk 2]
```diff
@@ -19,94 +19,141 @@ examples:
...
+              - !transform/shift-1.2.0
+                offset: 4.0
+        bounding_box: !transform/property/compound_bounding_box-1.0.0
+          selector_args:
+            - argument: x00
+              ignore: true
+            - argument: x10
+              ignore: true
+          cbbox:
+            - key: [0, 1] # both value of input x is 0 and value of input y is 1 to select this box
```
Comment (perrygreenfield) [inline resources/stsci.edu/schemas/property/compound_bounding_box-1.0.0.yaml:146]: Likewise.

**Review by perrygreenfield — COMMENTED** [review]
Looks good except for what looks like inconsistencies in the comments with the example details.

**Review by perrygreenfield — COMMENTED** [review]
LGTM

**Review by perrygreenfield — APPROVED** [review]
LGTM

**Comment by WilliamJamieson** [comment]
This will cause temporary CI failures. This is being coordinated with astropy/asdf-astropy#69, which will need to be released in order for the CI to function again.

**Pull request merged**